packages/dds/merge-tree/src/test/testClient.ts

```diff
@@ -206,7 +212,13 @@ export class TestClient extends Client {
 		overwrite?: boolean;
 		opArgs: IMergeTreeDeltaOpArgs;
 	}): void {
-		this.mergeTree.obliterateRange(start, end, refSeq, clientId, seq, overwrite, opArgs);
+		this.obliterateRangeRemote({
+			start,
+			end,
+			refSeq,
+			longClientId: "remote",
```

Comment:
@anthony-murphy - not sure if this is fine to do for tests or not. `obliterate.spec.ts` currently chooses arbitrary numbers to act as the "remote client id", but they don't have an associated longClientId because they don't actually exist. I also think all of the other obliterate test files use the reconnectHelper, so it doesn't affect them.

Reply:
i don't really understand, can you point me to a test where this is needed

Reply:
the testclient exposes the mergeTree, so if tests want to they can call the merge tree directly.

also, we should move this away from sequence place.